Sources/Shared/Promise.swift

```diff
@@ -27,7 +27,7 @@ extension Promise: ObservableConvertibleType {
 
 // MARK: - JSON dictionary
 
-extension Collection where Self: DictionaryLiteralConvertible, Self.Key == String, Self.Value: Any, Iterator.Element == (key: Self.Key, value: Self.Value) {
+extension Collection where Self: ExpressibleByDictionaryLiteral, Self.Key == String, Self.Value: Any, Iterator.Element == (key: Self.Key, value: Self.Value) {
```

Comment:
They are just type alias https://developer.apple.com/reference/swift/dictionaryliteralconvertible, but somehow the compiler shows some warnings